readme.md

```diff
@@ -17,8 +17,13 @@ var splitLines = require('split-lines');
 
 splitLines('foo\r\nbar\r\nbaz\nrainbow');
 //=> ['foo', 'bar', 'baz', 'rainbow']
+
+splitLines('foo\r\nbar\r\nbaz\nrainbow', true);
```

Comment:
Well technically `\n` is a newline. `\r` is a carriage return.

I do like the name better, and considered it. I kinda doubt anybody would be confused by it though.